LanguageFeatures/Inline-classes/dynamic_semantics_inline_class_A03_t02.dart

```diff
@@ -14,21 +14,91 @@
 
 import "../../Utils/expect.dart";
 
-inline class V {
+inline class V1 {
   final int id;
-  V(this.id);
+  V1(this.id);
+}
+
+inline class V2<T> {
+  final T id;
+  V2(this.id);
+}
 
-  String get foo => "Inline";
+String test(Object instance) {
```

Comment:
I think this one is a bit like a transposed matrix: We're testing that `o1 as T` and `o2 as T` will both throw or both succeed when `o1` is the value of an inline typed expression and `o2` is the value of an expression whose type is the instantiated representation type of that inline type, and we're doing that for a bunch of different values of `T`.

However, we should actually test that `o is V` and `o is R` for a bunch of different objects `o`, just like LanguageFeatures/Inline-classes/dynamic_semantics_inline_class_A03_t01.dart. (So we'd compare `o as V1` with `o as int`, and we'd compare `o as V2<String>` with `o as String`).

Basically, I think we should just have a copy of LanguageFeatures/Inline-classes/dynamic_semantics_inline_class_A03_t01.dart where it says `as` rather than `is`, and then we need to have some try/catch code around it in order to detect that we have "both are throwing" or "both succeed", not "one of each".

Let's try to extract what we can learn from this test: I think we're basically learning that `o1` and `o2` have the same run-time type (because they behave same with `o1 as T` and `o2 as T` for a set of different values of `T`). I think that's already covered by the direct test where we check that the run-time type of the value of an expression with an inline type is the representation type. So it should be fine to just change this test such that it's much more like LanguageFeatures/Inline-classes/dynamic_semantics_inline_class_A03_t01.dart.